tembo-cli/src/cmd/init.rs

```diff
@@ -42,11 +44,21 @@ pub fn execute() -> Result<(), anyhow::Error> {
         }
     }
 
-    let filename = "tembo.toml";
-    let filepath =
-        "https://raw.githubusercontent.com/tembo-io/tembo/main/tembo-cli/examples/single-instance/tembo.toml";
+    let cargo_manifest_dir =
+        env::var("CARGO_MANIFEST_DIR").expect("CARGO_MANIFEST_DIR env var not set");
```

Comment:
I think this logic only works because you are running from source, but would not work after build + distributing the binary

Reply:
You're right. I changed the code accordingly